rebar.config

```diff
@@ -17,6 +17,9 @@
 {edoc_opts, [{preprocess, true}]}.
 
 {profiles, [
+    {test, [
```

Comment:
It would be great to have this on Partisan and Lasp so we could run Erlang 20 builds there too...

Reply:
k